src/scripts/OSFramework/OSUI/Pattern/Tooltip/Tooltip.ts

```diff
@@ -59,10 +59,13 @@ namespace OSFramework.OSUI.Patterns.Tooltip {
 			this._tooltipBalloonPositionClass = this.configs.Position;
 		}
 
-		// Move balloon element to outside of the pattern context
+		// Move balloon element to active screen, outside of the pattern context
 		private _moveBalloonElement(): void {
-			const layoutElement = Helper.Dom.ClassSelector(document.body, GlobalEnum.CssClassElements.Layout);
-			Helper.Dom.Move(this._tooltipBalloonWrapperElem, layoutElement);
+			const activeScreenElement = Helper.Dom.ClassSelector(
```

Comment:
Is this method called multiple times? We should define the activeScreenElement screen only once. 
Maybe on the setHtmlElements method?

Reply:
However, to keep consistency, I've change the way that we are defining the element and do the same approach as Dropdown.